lifecycle-operator/chart/templates/lifecycle-manager-config.yaml

```diff
@@ -9,17 +9,17 @@ data:
   controller_manager_config.yaml: |
     apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
     health:
-      healthProbeBindAddress: {{ .Values.lifecycleManagerConfig.controllerManagerConfigYaml.health.healthProbeBindAddress
+      healthProbeBindAddress: {{ .Values.lifecycleManagerConfig.health.healthProbeBindAddress
```

Comment:
i would rename `.values.lifecycleManagerConfig` to `.values.lifecycleOperatorConfig`